Externals/glslang/SPIRV/GlslangToSpv.cpp

```diff
@@ -661,11 +661,11 @@ bool HasNonLayoutQualifiers(const glslang::TQualifier& qualifier)
 // Implement the TGlslangToSpvTraverser class.
 //
 
-TGlslangToSpvTraverser::TGlslangToSpvTraverser(const glslang::TIntermediate* glslangIntermediate, spv::SpvBuildLogger* buildLogger)
-    : TIntermTraverser(true, false, true), shaderEntry(0), sequenceDepth(0), logger(buildLogger),
+TGlslangToSpvTraverser::TGlslangToSpvTraverser(const glslang::TIntermediate* glslangIntermediate_, spv::SpvBuildLogger* buildLogger)
```

Comment:
this was for -Wshadow, accidentally made it in here since it's right next to a nullptr change. I can separate it out if need be